### PR TITLE
Implement delete button on front and back end

### PR DESCRIPTION
**Pull request by Lord-Bubbles**

### CODE REVIEW

File: src/controllers/RecipeController.js [hunk 1]
```diff
@@ -58,6 +58,12 @@ const addRecipe = asyncHandler(async (req, res) => {
     res.status(400);
     throw new Error('Body missing from request!');
   }
+  if (!verify(req.headers.authorization)) {
+    res.status(400);
+    throw new Error(
+      'Unauthorized access detected! Only the appropriate user can add this!'
```
Comment (chilimalone) [inline src/controllers/RecipeController.js:64]: 'Unauthorized access detected! Please log in before adding a recipe'?

File: src/controllers/RecipeController.js [hunk 2]
```diff
@@ -75,7 +81,21 @@ const addRecipe = asyncHandler(async (req, res) => {
...
-  // TODO: Backend delete button (Issue 35)
+  const { id } = req.params;
+  if (!id) {
+    res.status(400);
+    throw new Error('Recipe id missing from query parameters!');
+  }
+  if (!verify(req.headers.authorization)) {
+    res.status(400);
+    throw new Error(
+      'Unauthorized access detected! Only the appropriate user can delete this!'
```
Comment (chilimalone) [inline src/controllers/RecipeController.js:92]: 'Unauthorized access detected! Only the recipe owner can delete this'

**Review by chilimalone — APPROVED** [review]

File: src/controllers/RecipeController.js (hunk 1 above)
Comment (Lord-Bubbles) [reply src/controllers/RecipeController.js]: done

File: src/controllers/RecipeController.js (hunk 2 above)
Comment (Lord-Bubbles) [reply src/controllers/RecipeController.js]: done

**Pull request merged**